tests/incr_test.go

```diff
@@ -0,0 +1,55 @@
+package tests
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+)
+
+type tcase struct {
+	op  string
+	key string
+	val int64
+}
+
+func TestINCR(t *testing.T) {
+	var wg sync.WaitGroup
+	go runTestServer(&wg)
+	conn := getLocalConnection()
+
+	for _, tc := range []tcase{
+		{"s", "key1", 0},
+		{"i", "key1", 1},
+		{"i", "key1", 1},
+		{"i", "key2", 1},
+	} {
+
+		switch tc.op[0] {
+		case 's':
+			cmd := fmt.Sprintf("SET %s %d", tc.key, tc.val)
+			fireCommand(conn, cmd)
+		case 'i':
+			cmd := fmt.Sprintf("INCR %s", tc.key)
+			fireCommand(conn, cmd)
+		}
+	}
```

Comment:
This for loop is supposed to hold the test cases. You have only written the operations to be done and are checking the correctness at the end. This is not how unittests are written please go through other tests written in the codebase.

Reply:
@arpitbbhayani I've resolved the comments.. As I'm coming from java background Go syntax is new to me hope latest commits are fine